index.html

```diff
@@ -42,75 +42,44 @@ <h2><a href="#" class="action-text">LET’S CONNECT</a></h2>
     </section>
     <section class="works" id="portfolio">
         <div class="grid-container">
-            <div class="works-flex-container">
-                <img class="portfolio-item-img" src="img/Snapshoot Portfolio.png" alt="Tonic Website screenshot">
-                <div>
-                    <h2 class="portfolio-title">Tonic</h2>
-                <p class="history" id="bold-history">CANOPY</p>
-                <p class="history"><span><i class="fa-solid fa-circle fa-2xs"></i></span> Backend Dev</p>
-                <p class="history"><span><i class="fa-solid fa-circle fa-2xs"></i></span> 2015</p>
-                <p class="project-desc">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-                <ul id="portfolio-technologies">
-                    <li>HTML</li>
-                    <li>CSS</li>
-                    <li>JAVASCRIPT</li>
-                </ul>
-                <button class="see-demo-btn" type="button">See Project</button>
-                </div>
+        </div>
+    </section>
+    <div id="overlay-project" class="overlay-project">
+        <div class="works-flex-container-popup displayover">
+            <div class="flexbox-popup">
+                <h2 class="portfolio-title" id="popup-title">Tonic</h2>
+                <i class="fa-solid cancel fa-xmark" onclick="closeDemo()"></i>
             </div>
-
-            <div class="works-flex-container flex-flip">
-                <img class="portfolio-item-img" src="img/portfolio4.png" alt="Multi-Post Stories Website image">
-                <div>
-                    <h2 class="portfolio-title">Multi-Post Stories</h2>
+            <div class="categories"> 
                 <p class="history" id="bold-history">CANOPY</p>
-                <p class="history"><span><i class="fa-solid fa-circle fa-2xs"></i></span> Backend Dev</p>
-                <p class="history"><span><i class="fa-solid fa-circle fa-2xs"></i></span> 2015</p>
-                <p class="project-desc">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-                <ul id="portfolio-technologies">
-                    <li>HTML</li>
-                    <li>CSS</li>
-                    <li>JAVASCRIPT</li>
-                </ul>
-                <button class="see-demo-btn" type="button">See Project</button>
-                </div>
+            <p class="history" id="popup-history1"><span><i class="fa-solid fa-circle fa-2xs"></i></span> Backend Dev</p>
+            <p class="history" id="popup-history2"><span><i class="fa-solid fa-circle fa-2xs"></i></span> 2015</p>
             </div>
-
-            <div class="works-flex-container">
-                <img class="portfolio-item-img" src="img/tonic.png" alt="Facebook Website screenshot">
-                <div>
-                    <h2 class="portfolio-title">Facebook 360</h2>
```

Comment:
- [ ] From the project requirements, as the works section is **supposed to be rendered dynamically**, this means that HTML codes relating to it should be removed. I would suggest you remove all the codes here in the `works section`
👇 
![image](https://user-images.githubusercontent.com/90222110/195648606-8b567abc-da8a-4814-aca4-81f2f5131e41.png)